Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -205,6 +205,16 @@
 </style>
 <template>
   <div class="landing">
+    <b-row v-if="communication">

```

Comment:
Lets move this to a component, that way we separate its functionality from the landing